css/main.css

```diff
@@ -36,6 +36,8 @@ body {
     -webkit-flex: 1;
     -ms-flex: 1;
     flex: 1;
+    max-height: 100svh;
+    overflow-y: scroll;
```

Comment:
Rather than making the a fixed-height for the whole page which might hurt accessibility a bit, could you try to make the sidebar sticky? 🤔

Reply:
Okay I will do it that way